src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -1,4 +1,11 @@
 {
+  "IdentityOptions": {
```

Comment:
This isn't bound by configuration, so does nothing.
It would need to be in at least the `OrchardCore` section, and bound using `service.ConfigureOptions` in the users project to work

Reply:
https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity-configuration?view=aspnetcore-5.0#lockout

Reply:
The above would roughly work.

But because we don't bind any other identity options automatically from `appsettings.json` I think it would be easier, and better if we didn't provide automatic binding.

You have already written docs on how to do the binding https://docs.orchardcore.net/en/dev/docs/guides/password-configuration/ 

Why not just add it more docs there? and remove it from the `appsettings.json`

Reply:
Documentation added.